src/bundle/Resources/views/themes/admin/url_wildcard/list.html.twig

```diff
@@ -91,7 +93,7 @@
     {% endfor %}
 
     {% embed '@ibexadesign/ui/component/table/table.html.twig' with {
-        headline: 'url_wildcard.list.title'|trans|desc('URL wildcards'),
+        headline: custom_results_headline|default(results_headline(url_wildcards.getNbResults())),
```

Comment:
If customer want to extends or overwrite those headline he can just extends custom_results_headline variable
If not the default resutls_headline will be used.